src/utils/randomPhrase.ts

```diff
@@ -0,0 +1,385 @@
+const adjectives = [
```

Comment:
Just a small note, but it might be better to have these two lists in JSON files, so that we wouldn't have to clutter up this file. If you wan't to keep it as is that's also fine with me.

Reply:
I thought about this but it just seems like an extra step for not a lot of benefit. As I mentioned before, since this is likely to be write-once code, I think it's ok to leave a huge .ts file, rather than a short .ts file and two huge .json files.